apollo-router/src/apollo_studio_interop/mod.rs

```diff
@@ -24,6 +25,16 @@ use apollo_compiler::Schema;
 use router_bridge::planner::ReferencedFieldsForType;
 use router_bridge::planner::UsageReporting;
 
+/// This is a parallel enum to Configuration::ApolloSignatureNormalizationAlgorithm.
+// We need to use this because of the import method that the fuzzer uses. The fuzzer can't have apollo-router as
+// a direct dependency because that means the build crashes, so we use a dev dependency and do a path hack. This
+// means that the fuzzer can't import from the Configuration namespace and has to import this instead.
+#[allow(dead_code)]
```

Comment:
When I remove it I get the error `variant `Enhanced` is never constructed` even though it's obviously used below, I'm not sure if there's a better way to get around this rather than having the `#[allow(dead_code)]`?

Reply:
It's because of the fuzz import hack. The `Enhanced` variant is not constructed in the fuzzer I assume. If you clarify in the comment that that is why the `#[allow(dead_code)]` is required, it should be okay?